test/extensions/bootstrap/wasm/wasm_integration_test.cc

```diff
@@ -49,6 +49,10 @@ name: envoy.filters.http.wasm
       value: ""
     vm_config:
       vm_id: "my_vm_id"
+      environment_variables: 
+        host_env_keys: ["NON_EXIST"]
```

Comment:
We should add `getenv("NON_EXIST")` in tests to verify that it's not added as empty value.